src/js/components/Form/stories/HelpAndError.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 
-import { Box, Form, FormField, TextInput } from 'grommet';
+import { Box, Form, FormField, Text, TextInput } from 'grommet';
```

Comment:
For this story the help text on the second example was a little hard to recognize as help text and the error message has 2 icons which was confusing

Reply:
thats fair I can take out the icon in the second example